src/topojson/topojson.ml

```diff
@@ -416,17 +423,39 @@ module Make (J : Intf.Json) = struct
                     (decode_or_err (J.to_array (decode_or_err J.to_float)))))
               arcs
           in
+
+          (*let* transform =
+            (*J.to_string transform in
+            let float=
+               (J.to_list(J.to_array(decode_or_err J.to_float)))
+              transform*)
+            J.to_obj in let float= (J.to_list(J.to_array(decode_or_err (J.to_float J.to_float))))
+              transform*)
+          let* transform = J.to_obj transform in
+
+          let float = (J.to_array (decode_or_err J.to_float)) transform in
+
           let fm = foreign_members_of_json json in
-          Ok { objects = geometries; arcs; foreign_members = fm }
-      | _, _ -> Error (`Msg "No objects and/or arcs field in Topology object!")
+          Ok
+            {
+              objects = geometries;
+              arcs;
+              transform = float;
+              foreign_members = fm;
+            }
+      | _, _, _ ->
+          Error
+            (`Msg "No objects, arcs and/or transform field in Topology object!")
 
-    let to_json ?bbox { objects; arcs; foreign_members } =
+    let to_json ?bbox { objects; arcs; transform; foreign_members } =
       J.obj
         ([
            ("type", J.string "Topology");
            ( "objects",
              J.obj (List.map (fun (k, v) -> (k, Geometry.to_json v)) objects) );
            ("arcs", J.array (J.array (J.array J.float)) arcs);
+           (*("transform", ( (J.list(J.array J.float)) transform));*)
+           ("transform", J.obj (J.array J.float) transform);
```

Comment:
You'll need to convert `transform` to a `(string * json) list` encoding for example:

```json
{ "someKey": [ 1.0 ] }
```

Would be 

``ocaml
[ "someKey", `A [ `Float 1.0 ]]
```